dotnet/api/Presentation/Web/Controllers/Api/V1/ApiController.cs

```diff
@@ -26,5 +27,16 @@ public class ApiController : AndcultureCode.CSharp.Web.Controllers.ApiController
         public ApiController(IStringLocalizer localizer) : base(localizer) { }
 
         #endregion Constructors
+
+        #region Public Methods
+
+        /// <summary>
+        /// Responds with HTTP 200 Ok
+        /// TODO: Abstract to AndcultureCode.CSharp.Web.Controllers.Controller
```

Comment:
Waiting on PR https://github.com/AndcultureCode/AndcultureCode.CSharp.Web/pull/8 and I'll remove this change before merging.